src/mito2/src/access_layer.rs

```diff
@@ -147,9 +148,16 @@ impl AccessLayer {
             }
             .build();
             let mut writer = ParquetWriter::new(
-                file_path,
+                || async {
+                    self.object_store
+                        .writer_with(&file_path)
+                        .chunk(DEFAULT_WRITE_BUFFER_SIZE.as_bytes() as usize)
+                        .concurrent(DEFAULT_WRITE_CONCURRENCY)
+                        .await
```

Comment:
Is it possible to wrap a helper function to create the writer? We repeat this many times.